locations/newyorck.yml

```diff
@@ -189,3 +195,5 @@ location__wireless_profiles__to_merge:
         network: prdhcp
         radio: [11a_standard, 11g_standard]
         ifname_hint: prdhcp
+
+flow_offload: hw
```

Comment:
Move this to the individual host entry further above. Flow offloading has issues on some SoCs, we don't want surprises in the future when device models in this location change.